source/includes/examples/tf-example-third-party-integration-stagingprod.rst

```diff
@@ -0,0 +1,28 @@
+.. code-block:: shell 
+   :copyable: true 
+
+   resource "mongodbatlas_third_party_integration" "test_third_party_integration" {
+     project_id = var.atlas_project_id
```

Comment:
remove atlas_?

Reply:
this is how it is set in the variables file.